### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by yochannah**
au vs nz, plus clarified one sentence

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The OLS Governance Committee (referred to as ‘Committee’ hereafter) is a ded
 * Hao Ye, University of Pennsylvania / Community for Rigor, USA
 * Laura Ación, MetaDocencia, Argentina 
 * Laurah Ondari, Bioinformatics Hub of Kenya initiative (BHKi), Kenya
-* Rowland Mosbergen, Practical Diversity and Inclusion Project, New Zealand 
+* Rowland Mosbergen, Practical Diversity and Inclusion Project, Australia
```
Comment (malvikasharan) [inline README.md:25]: Grave mistake! Spotted by a Kiwi.

**Review by malvikasharan — APPROVED** [review]
Thank you.

File: README.md (hunk 1 above)
Comment (NPDebs) [reply README.md:25]: I knew something was fishy. 👀 I have to quickly change it in the main governance page.

Comment (yochannah) [reply README.md:25]: Malvika's basically saying she prefers ghanain jollof rice here...

**Pull request merged**